esmvaltool/diag_scripts/magic_bsc/diurnal_temp_index.r

```diff
@@ -0,0 +1,204 @@
+library(yaml)
+library(s2dverification)
+library(multiApply) # nolint
+library(climdex.pcic)
+library(ClimProjDiags) #nolint
+library(parallel)
+library(ncdf4)
+source("/home/Earth/nperez/git/ClimProjDiags/R/DTRRef.R") #nolint
```

Comment:
```
Loading required package: maps

Attaching package: ‘s2dverification’

The following object is masked from ‘package:base’:

    Filter

Loading required package: PCICt

Attaching package: ‘ClimProjDiags’

The following object is masked from ‘package:s2dverification’:

    Subset

Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
Calls: source -> file
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  cannot open file '/home/Earth/nperez/git/ClimProjDiags/R/DTRRef.R': No such file or directory
Execution halted
```

Reply:
Dear @bouweandela, apologies for this mistakes. 

Line 8, source("/home/Earth/nperez/git/ClimProjDiags/R/DTRRef.R") #nolint, is not needed. This function is already included in the ClimProjDiags. 

I'm trying to  run the recipe in my home without this line to check if everything works. However, esmvaltool returns error because _main.py has been modify. If I correct this error, then the error is

from prov.dot import prov_to_dot
ModuleNotFoundError: No module named 'prov'

What can I do? Should I remove this line number 8? 

Thanks!

Reply:
> However, esmvaltool returns error because _main.py has been modify. If I correct this error

Can you be more specific about the error? What is the error message? There should be no need to modify esmvaltool/_main.py.

> then the error is

>from prov.dot import prov_to_dot
ModuleNotFoundError: No module named 'prov'

I would recommend running `pip install -e .[develop]` again as described in the README.md, because the Python dependencies have been updated because additional packages are needed for provenance logging. That should solve the ModuleNotFoundError.

Reply:
Thanks, @bouweandela! Everything is working now.

Reply:
This is probably related to having multiple versions of GCC installed, possibly in your conda environment, one of them without support for OpenMP 4.0, see e.g. [this link](https://stackoverflow.com/questions/39017485/openmp-linux-gomp-4-0-not-found).

I would recommend making sure your OS has a recent version of GCC installed (i.e. use your OS's package manager or if you're on a managed system use `module avail/module load` to set it up) and creating a new clean installation following the instructions in README.md, i.e. make a new conda environment, etc.